src/TodoListItem.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+// import TodoList from "./TodoList";
+
+
+
+const TodoListItem = (props) => {
+    return(
+        <li key={props.key}> {props.todo} </li>
```

Comment:
because we renamed todo as title in TodoList component , here we also need to update it bc todo is not passed anymore

`<li key={props.key}> {props.title} </li>`